src/controllers/gym.js

```diff
@@ -178,6 +178,8 @@ class Gym extends Controller {
 
 					require('./common/nightTime').setNightTime(data, conqueredTime)
 
+					data.intersection = await this.getIntersection(data.latitude, data.longitude);
```

Comment:
looks like getIntersection returns an array... how are you expecting the DTS to be used?

Reply:
Was using an array for WDR, just changed it to return a string so it should be good.